packages/shared/src/helpers/Config.ts

```diff
@@ -0,0 +1,32 @@
+import Big from "big.js";
+
+export const MILLISECONDS_PER_WEEK = Big(1000 * 60 * 60 * 24 * 7);
+
+/*
+ * Returns the appropriate Raha network config value.
+ * Currently the only condition ever used is time,
+ * but this could be extended to include transaction
+ * history or external data sources.
+ */
+export class Config {
+  public static readonly UBI_WEEKLY_RATE = Big(10)
+  public static readonly MAX_WEEKS_ACCRUE = Big(4)
+  public static readonly REFERRAL_BONUS_PRE_SPLIT = Big(60);
+  public static readonly REFERRAL_BONUS_POST_SPLIT = Big(30);
+  public static readonly INVITED_BONUS_PRE_SPLIT = Big(0);
+  public static readonly INVITED_BONUS_POST_SPLIT = Big(30);
+  public static readonly DEFAULT_DONATION_RATE = Big(0.03);
+  public static readonly REFERRAL_SPLIT_DATE = Date.UTC(2019, 1, 1);
+  public static get REFERRAL_BONUS() {
+    if (Date.now() < this.REFERRAL_SPLIT_DATE) {
```

Comment:
Should this also depend on member created value? Otherwise might suck for people who didn't mint their referral bonuses beforehand

Reply:
I'd prefer to be consistent with what we did for enforcing the 40 Raha Mint cap, and if you're too late it's simply too late. That way it's always same rules for everyone at a given time, and we don't have to maintain different legacy rules for older members/invites in the codebase.

Reply:
I hear you about not maintaining legacy rules, but that means we have some people who get 30 Raha referral bonus but their friend still doesn't get anything. If we're okay with that I guess it's fine. Does that mean we should try to send out notifications before the transition then?

Reply:
We can check on how many people this will effect and if needed notify just them ... have already sent out 2 email warnings.

Reply:
I can check and notify